css/style.css

```diff
@@ -0,0 +1,89 @@
+body {
```

Comment:
[ OPTIONAL ]There is a horizontal overflow which means some content is overflowing out of the container. Please fix the overflow as it's not considered a good design.